Module/Module.xctemplate/View/___FILEBASENAME___Presenter.swift

```diff
@@ -12,12 +12,7 @@ class ___VARIABLE_productName___Presenter: Presentable {
     var actions: ___VARIABLE_productName___Actionable
     var parameters: ___VARIABLE_productName___Parameterable
     
-    // MARK:- Typealias
-    
-    typealias Actionable = ___VARIABLE_productName___Actionable
-    typealias Parameterable = ___VARIABLE_productName___Parameterable
-    typealias Viewable = ___VARIABLE_productName___Viewable
-    
+    // MARK:- Init
```

Comment:
@aaronTredrea other templates uses `Initialiser` :)

Reply:
hahaha, I saw that just as I committed and thought - hmmm, hopefully no one will notice! lol